src/tx-format/transaction.md

```diff
@@ -83,7 +83,11 @@ Transaction is invalid if:
 - `scriptDataLength > MAX_SCRIPT_DATA_LENGTH`
 - `scriptLength * 4 != len(script)`
 - `scriptDataLength != len(scriptData)`
-- `gasLimit` is less than the sum of all `predicateGasUsed` for `InputType.Coin` or `InputType.Message` where predicate length is greater than zero.
+- No policy of type `PolicyType.GasPrice`
+- No policy of type `PolicyType.GasLimit`
```

Comment:
I think we can allow not have`PolicyType.GasLimit` in the case of an empty `script`.

Reply:
I just made gas limit a default field for script transactions. The only reason to keep it as a policy is to save a u64 off of basic transfers that don't use any script logic.